Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp

```diff
@@ -708,13 +708,13 @@ namespace DRUtils
             return FoundMeshComp;
         }
 
-        TArray<UActorComponent*> ChildMeshComps = OwnerActor->GetComponentsByClass(UMeshComponent::StaticClass());
-        for (UActorComponent* CheckComp : ChildMeshComps)
+        TArray<UMeshComponent*> ChildMeshComps;
+        OwnerActor->GetComponents<UMeshComponent>(ChildMeshComps);
+        for (auto CheckComp : ChildMeshComps)
```

Comment:
You can rename CheckComp to CheckMeshComp here, and keep the code inside the same

Reply:
done